android/app/src/main/java/com/papel/ui/articles/ReadArticleActivity.java

```diff
@@ -96,6 +99,34 @@ protected void onCreate(Bundle savedInstanceState) {
         cl_black = ColorStateList.valueOf(getResources().getColor(R.color.black));
         getArticleFromEndpoint(getApplicationContext(), articleId);
 
+
+        final Intent profileIntent = new Intent(this, ProfileActivity.class);
+        // TODO This should be inactive during the request to server
+        author.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                profileIntent.putExtra("UserId",authorId);
+                startActivity(profileIntent);
+            }
+        });
+
+        profile_pic.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                profileIntent.putExtra("UserId",authorId);
+                startActivity(profileIntent);
+            }
+        });
+
```

Comment:
These lines really solves my problem on the following functionality --> 
_Accessing other profiles via clicking on the comments of the article OR the author of the article_